apps/prairielearn/src/components/QuestionScore.html.ts

```diff
@@ -275,7 +275,7 @@ export function ExamQuestionStatus({
   const badge_color = {
     unanswered: 'warning',
     invalid: 'danger',
-    grading: 'default',
+    grading: 'info',
```

Comment:
This is a weird one, since `badge-default` did not exist in BS4 either...

Reply:
> When does this badge end up in this state? `info` has sort of conventionally come to be associated with manual grading. I'm not sure if that's the most appropriate color here.

This status comes up for external grading between sending the submission and the grading completing. To more easily test it choose a question with slow autograder and/or submit code with sleep calls.

Reply:
Ok, that's interesting, this status is indeed never used in the codebase that I could find... I added a comment.

> `info` has sort of conventionally come to be associated with manual grading. I'm not sure if that's the most appropriate color here.

Actually, "saved for manual grading" is assigned "success" a few lines before this. "info" is used for saved status.

Reply:
To be clear: the "grading" status shows up as the computed status of a grading job, but not as the status of the instance question. We could add a custom validation here to check if there is a pending grading job and show the grading status, but I think the gain would be marginal for the complexity, and if needed it should probably be done in a separate PR.

@nwalters512 this is approved, but you commented after the approval. Can I proceed or do you want to review it as well?

Reply:
I did some extra checks, and from what I could gather the grading status was removed in #5909. I'm now wondering if this removal was unintential as a side-effect of removing the points_in_grading update sproc. The original meaning, from what I could gather, is what I mentioned above: grading has been triggered, but has not yet completed. Is there any value in restoring this status? Should be relatively simple to do (just add an update to the [grading job insertion](https://github.com/PrairieLearn/PrairieLearn/blob/c3c2f592a6b3c4c93ca09c403ae6309481fcd829/apps/prairielearn/src/models/grading-job.ts#L43-L69) process, which was the point where this was originally updated at), and given the volatile nature of this status it would likely not require a migration for historical data.

Reply:
Posted as a new issue in #12338

Reply:
Thanks for the digging! I approved that one.

I think I'd prefer to use `secondary` here; that matches the indicator that's shown on a submission when external grading is in progress. Does that reasoning make sense?

Reply:
Makes sense. Done in 5c2caca65da59e815c61757b2e6b7fd97e7b6e40.